fvm/environment/system_contracts.go

```diff
@@ -264,6 +269,7 @@ var accountStorageCapacitySpec = ContractFunctionSpec{
 	ArgumentTypes: []sema.Type{
 		&sema.AddressType{},
 	},
+	UseVM: true,
```

Comment:
I would not do:
- accountStorageCapacity
- accountBalance
- accountAvailableBalance

because they are user invoked functions.

But there is one missing: accountsStorageCapacity on line ~300 (accounts vs account). I would do that one because that is only used for bookkeeping.

Reply:
Sounds goof! I've disabled the VM for the three listed specs in [0c4763e](https://github.com/onflow/flow-go/pull/7369/commits/0c4763e836a3ed7f056c0e9c004613110a3ae1cd).

We could enable `accountsStorageCapacity`, as the compiler/VM seems to implement enough to make it work, but the `TestSettingExecutionWeights` fails. I guess the assertions fail because the VM does not meter computation yet?

Reply:
yes `accountsStorageCapacity` is metered, but fees and payer check are not. If the `compiler/VM` doesn't support metering yet, we will have to skip that one.